appinventor/components/src/com/google/appinventor/components/runtime/ChartData2D.java

```diff
@@ -40,6 +50,7 @@ public ChartData2D(Chart chartContainer) {
     dataFileColumns = Arrays.asList("", "");
     sheetsColumns = Arrays.asList("", "");
     webColumns = Arrays.asList("", ""); // Construct default webColumns list with 2 entries
```

Comment:
See note on AnomalyDetection about moving this initialization into the DataCollection class.